src/ui/Typography/index.tsx

```diff
@@ -22,7 +22,7 @@ type TypographyVariant =
   | 'underline'
   | 'link'
 
-type TypographyColor =
+export type TypographyColor =
```

Comment:
You export this but it seems like you don't use it. Is that on purprose?

Reply:
✅